worker-message-prioritization-rerouting/src/main/java/com/github/workerframework/workermessageprioritization/rerouting/MessageRouterSingleton.java

```diff
@@ -81,10 +89,10 @@ public static void init() {
             messageRouter = new MessageRouter(queuesApi, "/", stagingQueueCreator, rerouteDecider);
         }
         catch (final Throwable e) {
-            LOGGER.error("Failed to initialise WMP - {}", e.toString());
+            initError = String.format("Failed to initialise WMP - %s", e);
```

Comment:
I am wondering if should we change the behavior of init?

Currently, if a worker calls init and there is an exception, we log an error, and the call to route returns without doing anything, i.e. the message is sent to the original staging queue. So, it's not really obvious there is a problem unless someone is actively looking at the logs.

As we adding a healthcheck now that will report unhealthy if init fails, or we cannot connect to the management api, should init instead just rethrow the exception after logging it?